### PR TITLE
EUI-112 pending org details

### PR DESCRIPTION
**Pull request by toireasam**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/org-manager/org-manager.module.ts [hunk 1]
```diff
@@ -10,17 +9,14 @@ import * as fromContainers from './containers';
...
 import { EffectsModule } from '@ngrx/effects';
 import { reducers, effects } from './store';
 import { OrgOverviewComponent } from './containers/org-overview/org-overview.component';
 import { OrgSummaryComponent } from './containers/org-summary/org-summary.component';
 import { OrgPendingModule } from 'src/org-pending/org-pending.module';
 import { NotificationBannerComponent } from 'src/org-manager/components/notification-banner/notification-banner.component';
-import { IdentityBarComponent } from 'src/org-manager/components/identity-bar/identity-bar.component';
-
-export const COMPONENTS = [ OrgOverviewComponent, OrgSummaryComponent, NotificationBannerComponent, IdentityBarComponent];
+export const COMPONENTS = [ OrgOverviewComponent, OrgSummaryComponent, NotificationBannerComponent ];
```
Comment (fazanki) [inline src/org-manager/org-manager.module.ts]: can you please add barelling index.ts

File: src/org-manager/store/effects/single-org.effects.ts [hunk 2]
```diff
@@ -22,7 +22,7 @@ export class SingleOrgEffects {
       console.log('LOAD_SINGLE_ORGANISATION ::: data is', data);
       return this.orgService.getSingleOrganisation(data.payload).pipe(
         map(singleOrgDetails => {
-          console.log('singleOrganisationDetails ===>', singleOrgDetails);
+          console.log('singleOrganisationDetails', singleOrgDetails);
```
Comment (fazanki) [inline src/org-manager/store/effects/single-org.effects.ts]: can you remove console.log

File: src/org-pending/components/summary/summary.component.html [hunk 3]
```diff
@@ -0,0 +1,78 @@
+<app-identity-bar-component><span>{{data.name}} </span><span class="hmcts-badge">{{data.status}}</span></app-identity-bar-component>
+<div class="govuk-width-container">
+<div class="govuk-main-wrapper">
+<div class="govuk-heading-xl">
```
Comment (fazanki) [inline src/org-pending/components/summary/summary.component.html]: <div class="govuk-width-container">
<div class="govuk-main-wrapper"> 
should live in the compoents that wrapps compoent with ng-content

File: src/org-pending/containers/org-pending-summary/org-pending-summary.component.ts [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+  constructor(
+    private store: Store<fromOrganisationPendingStore.PendingOrganisationState>) { }
+
+  ngOnInit() {
+
+    this.orgSummary$ = this.store.pipe(select(fromOrganisationPendingStore.getSingleOrgOverview));
+    this.loading$ = this.store.pipe(select(fromOrganisationPendingStore.orgSummaryLoading));
+
+    this.$routeSubscription = this.store.pipe(select(fromOrganisationPendingStore.getCurrentPage)).subscribe((routeParams) => {
+      if (routeParams.id && routeParams.id !== this.pageId) { // TODO see why double call.
```
Comment (fazanki) [inline src/org-pending/containers/org-pending-summary/org-pending-summary.component.ts]: can u please unsbuscribe this on ng destroy

File: src/org-pending/containers/overview/pending-overview.component.ts [hunk 5]
```diff
@@ -47,6 +44,7 @@ processCheckedOrgs(pendingOrgs) {
     console.log('in pending checked orgs are', pendingOrgs.value);
     // TO DO DISPATCH AN ACTION ETC HERE
```
Comment (fazanki) [inline src/org-pending/containers/overview/pending-overview.component.ts]: remove console log

File: src/org-manager/org-manager.module.ts (hunk 1 above)
Comment (toireasam) [reply src/org-manager/org-manager.module.ts]: done

File: src/org-pending/components/summary/summary.component.html (hunk 3 above)
Comment (toireasam) [reply src/org-pending/components/summary/summary.component.html]: done

File: src/org-pending/containers/org-pending-summary/org-pending-summary.component.ts (hunk 4 above)
Comment (toireasam) [reply src/org-pending/containers/org-pending-summary/org-pending-summary.component.ts]: done

File: src/org-pending/containers/overview/pending-overview.component.ts (hunk 5 above)
Comment (toireasam) [reply src/org-pending/containers/overview/pending-overview.component.ts]: done

**Pull request merged**